### DISCUSSION TITLE
How to get the velocity of points at rotor plane but not within the swept area?

### DISCUSSION THREAD

**Discussion by Darry6682** [NatLabRockies/floris#923] [Q&A]
Hi @rafmudaf Thanks for your great efforts making the FLORIS so powerful! I noticed that the TurbineGrid can only get the velocity at the rotor plane within the swept area, However, is there any method to obtain the nine-point profile for each WT as shown in this figure? This is important for my following research. Appreciate it if you could help!
![image](https://github.com/NREL/floris/assets/71394037/da1c2dfc-7f89-490e-bfda-c9f51d1a3a47)

**Comment by misi9170** [comment]
Hi @Darry6682 ,

I'm sorry that we didn't respond to this sooner---it somehow slipped past me. You can sample arbitrary points in the flow using [`FlorisModel.sample_flow_at_points()`](https://nrel.github.io/floris/_autosummary/floris.floris_model.html#floris.floris_model.FlorisModel.sample_flow_at_points), which accepts lists of (x,y,z) coordinates where you could like to extract the flow velocities. 

See [this example](https://nrel.github.io/floris/examples/examples_get_flow/002_extract_wind_speed_at_points.html#) for usage.

Hope this helps, and again, I'm sorry for the slow reply here.
